mapeditor/Animation.cpp

```diff
@@ -0,0 +1,780 @@
+/*
+ * CAnimation.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#include "StdInc.h"
```

Comment:
Is it a new file? Don't we have it for client already? Or is it same animation but slightly modified for QT?

Reply:
a lot of duplicated code there to render graphics using QT instead of SDL. I tried do not touch that much